src/http-api/interface.ts

```diff
@@ -61,6 +61,8 @@ export interface IRequestOpts {
     // This is intended for use on endpoints where M_UNKNOWN_TOKEN is a valid/notable error response,
     // such as with token refreshes.
     inhibitLogoutEmit?: boolean;
+
+    priority?: RequestInit["priority"];
```

Comment:
doc-comment please. "Priority for the request, in browsers which support it. See https://html.spec.whatwg.org/multipage/urls-and-fetching.html#fetch-priority-attribute".

Or something to that effect